### PR TITLE
Add reassign feature to Project Tasks

### PR DESCRIPTION
**Pull request by dreispt**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
You might use generic OCA's module icon, but :+1:

File: project_task_reassign/README.rst [hunk 1]
```diff
@@ -0,0 +1,38 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
```
Comment (yvaucher) [inline project_task_reassign/README.rst:8]: Please put the name of the module as title

File: project_task_reassign/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    'name': 'Reassign Project Task',
+    'summary': 'Reassign Tasks to other Responsibles and Projects',
```
Comment (yvaucher) [inline project_task_reassign/__openerp__.py:20]: missing license key

File: project_task_reassign/README.rst [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+Regular users (Employee group) will be able to see who is handling the request,
+but won't be able to change it.
+
+Project Users can click on the "=> Reassign" link, in front of the current
+responsible, to open a dialog where they can select the new responsible and/or
+new Project/Service Team it should be assigned to.
+
+Mass reassignments can also be made, through the context menu action on the
+list view.
+
```
Comment (yvaucher) [inline project_task_reassign/README.rst:21]: You should add the bug tracker link

File: project_task_reassign/project.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2013 Daniel Reis
```
Comment (yvaucher) [inline project_task_reassign/project.py]: I think copyright year needs an update AFAIK you didn't created a v8.0 module in 2013 :)

File: project_task_reassign/wizard/project_task_reassign_view.xml [hunk 5]
```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+
+        <record id="view_project_task_reassign" model="ir.ui.view">
+            <field name="name">Reassign Project Task</field>
+            <field name="model">project.task.reassign</field>
+            <field name="arch" type="xml">
+
+                <form string="Reassign Project Task" version="7.0">
```
Comment (yvaucher) [inline project_task_reassign/wizard/project_task_reassign_view.xml]: version="7.0" is obsolete

**Comment by rafaelbn** [comment]
@dreispt Let me know when ready for testing, thanks!

**Comment by pedrobaeza** [comment]
Is this ready for review? Maybe you can add tests for keeping a good coverage.

**Comment by dreispt** [comment]
@pedrobaeza I guess it is. If you don't mind Pedro, I would rather add tests when migration this to 9.0.

**Comment by pedrobaeza** [comment]
OK, as you wish, but tests can be the same for both versions (if you don't use demo data, as my team always do), and even help on the migration task.

**Comment by dreispt** [comment]
This PR is two years old and still old API...

**Comment by pedrobaeza** [comment]
OK, you can make the test in new API without problems, but I don't insist more, hehe.

**Review by pedrobaeza — APPROVED** [review]
OK, we can merge this one as this is for v8 and improve coverage on v9.

**Comment by dreispt** [comment]
Thanks!

**Pull request merged**